frb_codegen/src/generator/rust/ty_delegate.rs

```diff
@@ -58,10 +57,19 @@ impl TypeRustGeneratorTrait for TypeDelegateGenerator<'_> {
             IrTypeDelegate::ZeroCopyBufferVecPrimitive(_) => {
                 Acc::distribute(Some("ZeroCopyBuffer(self.wire2api())".into()))
             },
-            IrTypeDelegate::StringList => Acc {
-                io: Some(TypeGeneralListGenerator::WIRE2API_BODY_IO.to_owned()),
-                wasm: Some(TypeGeneralListGenerator::WIRE2API_BODY_WASM.to_owned()),
-                ..Default::default()
+            IrTypeDelegate::StringList =>{
+                let x = self.ir.get_delegate();
+                let prefix = self.get_wire2api_prefix(&x);
+                // REPLACE PREFIX TO WIRE2API_BODY_IO AND WIRE2API_BODY_WASM
+                let wire2api_body_io =
+                    TypeGeneralListGenerator::WIRE2API_BODY_IO.replace("Wire2Api", &prefix);
+                let wire2api_body_wasm =
+                    TypeGeneralListGenerator::WIRE2API_BODY_WASM.replace("Wire2Api", &prefix);
+                Acc {
+                    io: Some(wire2api_body_io),
+                    wasm: Some(wire2api_body_wasm),
+                    ..Default::default()
+                }
```

Comment:
nit: again, this `REPLACE PREFIX TO WIRE2API_BODY_IO AND WIRE2API_BODY_WASM` seems to happen twice (here, and in another place), so maybe we can extract